studio-docs/source/api-keys.mdx

```diff
@@ -10,22 +10,29 @@ Every application that sends data to Apollo must use an **API key** to do so. Ap
 
 ## Graph API keys
 
-A graph API key provides **full** access to a **single graph** in Apollo Studio. A GraphQL server (such as Apollo Server) can use a graph API key to register its schema and push metrics to Studio.
+A graph API key provides access to a **single graph** in Apollo Studio. A GraphQL server (such as Apollo Server) can use a graph API key to register its schema and push metrics to Studio.
 
 Create a unique graph API key for each non-development system that communicates with Studio. Doing so enables you to revoke access to a single system without affecting others.
 
 <ObtainGraphApiKey />
 
+### Setting permissions
+
+Unless you have an Enterprise plan, every graph API key provides **full access** to its associated graph.
+
+If you have an Enterprise plan, you can [assign a member role](/org/members/#member-roles) to each graph API key you create. If you do, the API key's permissions are limited to that role's permissions.
+
+You can't change a graph API key's role after it's created. Instead, create a new key with the desired role.
+
 ## Personal API keys
 
-A personal API key provides **partial** access to **every graph in every organization you belong to**. Specifically, the Apollo [CLI](https://www.apollographql.com/docs/devtools/cli/) and [VS Code extension](https://www.apollographql.com/docs/devtools/editor-plugins/) support personal API keys, but all other Studio interactions require a graph API key.
+A personal API key provides partial access to **every graph in every organization you belong to**. A personal API key has the same permissions that your user account has in each of your organizations. 
 
-Interactions that require a graph API key include:
+You can use personal API keys with local development tools (like the Apollo [CLI](https://www.apollographql.com/docs/devtools/cli/) and [VS Code extension](https://www.apollographql.com/docs/devtools/editor-plugins/)) to load schemas and other data from Studio.
+
+However, a [graph API key](#graph-api-keys) is required for almost all other Studio interactions, including:
```

Comment:
"almost all" seems like a real stretch here. It's really just those two things (and actually technically, schema reporting can work with user keys, but the Apollo Server implementation doesn't support it). And op reg but I'm uninterested in making things more complex by mentioning its existence.

I would say that personal API keys don't work with Apollo Server, basically?